### DISCUSSION TITLE
Consumer group has strange members when kafka uses ingress external listener

### DISCUSSION THREAD

**Discussion by vutkin** [strimzi/strimzi-kafka-operator#5171] [Q&A]
Hi @scholzj , 

I see that out consumer group has strange members when kafka uses ingress external listener (nginx by the way):
3 fake members for 1 pod with consumer:

topic: teamname.test (10 partitions)
consumer group: test-service

consumer-test-service-3 | consumer-test-service-3-f36d7a39-aadf-4340-a153-7eae4f70b9ad | X-16-0-85.nginx-ingress-controller-admission.kube-system.svc.cluster.local/X.16.0.85 | teamname.test (8)

consumer-test-service-3 | consumer-test-service-3-b04dbe1a-86d4-495d-9bfe-1e375e88d993 | X-16-3-182.nginx-ingress-controller-admission.kube-system.svc.cluster.local/X.16.3.182 | teamname.test (7)
consumer-test-service-3 | consumer-test-service-3-030b1552-34e6-41eb-9236-906bfc2e3b14 | X-16-0-85.nginx-ingress-controller-admission.kube-system.svc.cluster.local/X.16.0.85 | teamname.test (6)

Will these leads to problems with reading messages, because 2 of these consumers in group are fakes and looks like were created by prometheus request and nginx admission controller?

Do anyone saw same behaviour with ingress type external listener?

**Comment by scholzj** [comment]
TBH, I'm not sure I follow what are the outputs above and where do they come from. I do not think the ingress listener can create any consumer groups ... it is just listener a listener as any other with the advertised address based on the Ingress address. To create some consumer group (or members in existing one), that has to be bound to a Kafka client running somewhere.
